docs/standards/universal-profile/01-lsp0-erc725account.md

```diff
@@ -41,19 +41,44 @@ See the section **["Members of address types"](https://docs.soliditylang.org/en/
 
 :::
 
-This substandard enables the account to execute generic calls on any other smart contracts, including transferring native tokens along with the call. This is made possible via a generic [`execute(...)`](../smart-contracts/lsp0-erc725-account.md#execute) function in the smart contract. **Only the owner can execute** the operations below.
+This substandard enables the account to execute generic calls on any other smart contracts, including transferring native tokens along with the call. External actions are possible via a smart contract's generic [`execute(...)`](../smart-contracts/lsp0-erc725-account.md#execute) function in the smart contract. **Only the owner can perform** the operations below.
 
 The ERC725X standard also enables deploying new smart contracts by providing the bytecode of the new contract to deploy as an argument to the `execute(...)` function. Contracts can be deployed using either CREATE or [CREATE2](https://eips.ethereum.org/EIPS/eip-1014).
 
 The following types of calls (= operation types) are available:
 
-| Operation number |                     Operation type                     | Description                                                                                                                             |
-| :--------------: | :----------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------- |
-|        0         |                         `CALL`                         | call another smart contract                                                                                                             |
-|        1         |                        `CREATE`                        | create a new smart contract with the associated bytecode passed as `_data`                                                              |
-|        2         |  [`CREATE2`](https://eips.ethereum.org/EIPS/eip-1014)  | create a new smart contract with a **salt **(for pre-computed contract addresses)                                                       |
-|        3         | [`DELEGATECALL`](https://eips.ethereum.org/EIPS/eip-7) | run the function from another contract, but use and update the storage of the current contract (= persist `msg.sender` and `msg.value`) |
-|        4         | [`STATICCALL`](https://eips.ethereum.org/EIPS/eip-214) | call another smart contract while disallowing any modification to the state during the call                                             |
+<table>
```

Comment:
Please keep the tables as markdown, as we agreed to not use HTML tables in docs.